### ISSUE TITLE
Update `outline: none` per WCAG

### ISSUE THREAD

**Issue by ntwb** [WordPress/twentytwenty#558]
Defining focus to navigation elements is an accessibility requirement, it's clearly stated in the [Web Content Accessibility Guidelines](https://www.w3.org/TR/2008/REC-WCAG20-20081211/#navigation-mechanisms-focus-visible):

>  2.4.7 Focus Visible: Any keyboard operable user interface has a mode of operation where the keyboard focus indicator is visible. (Level AA)

See also http://www.outlinenone.com/

There are two instances where `outline: none;` are used and should be updated:

https://github.com/WordPress/twentytwenty/blob/eec0bc0d4d89a5a7ee6aab1509164f516504890a/style.css#L1636

https://github.com/WordPress/twentytwenty/blob/eec0bc0d4d89a5a7ee6aab1509164f516504890a/style.css#L1873

This is part of a pull request to follow adding the `a11y/no-outline-none` [stylelint-a11y rule](https://github.com/YozhikM/stylelint-a11y/tree/master/src/rules/no-outline-none) to check for accessibility issues

**Comment by nielslange** [comment]
@ntwb This issue should be solved with #548
